providers/ns1/ns1Provider.go

```diff
@@ -302,7 +302,7 @@ func buildRecord(recs models.Records, domain string, id string) *dns.Record {
 		if r.Type == "MX" {
 			rec.AddAnswer(&dns.Answer{Rdata: strings.Fields(fmt.Sprintf("%d %v", r.MxPreference, r.GetTargetField()))})
 		} else if r.Type == "TXT" {
-			rec.AddAnswer(&dns.Answer{Rdata: r.GetTargetTXTSegmented()})
+			rec.AddAnswer(&dns.Answer{Rdata: []string{r.GetTargetField()}})
```

Comment:
I've been meaning to write up about this new function and when to use it.  For the meanwhile... trust me :-)

```suggestion
			rec.AddAnswer(&dns.Answer{Rdata: []string{r.GetTargetTXTJoined()}})
```

Reply:
sorry now coming to that - sure :)